### PR TITLE
Fix broken code in custom latent trajectory example

### PR DESCRIPTION
**Pull request by PlethoraChutney**
Working with Hannah, we found a few parts of this example that do not work. This version fixes those regions. Not sure if the fixed version of the dataset saving is still the right way to go though, was this written before `save_external_result()`?

### CODE REVIEW

File: docs/examples/3dflex-custom-latent-trajectory.ipynb [hunk 1]
```diff
@@ -247,42 +220,57 @@
...
...OI0aBoA+Da7OaJ3DIiwbGISdr371q9XhjGDNYQuYXGzZVHWFR5BDIDIxEN8D/uI1QAh68boZLYpwTJDc5eLe5RVQZ0Hn7nWt3rqP2zY0G5eOezCCDzQCNhCUBLDHH398JSB+5zvfqZLGmTRIyzb16FZufDt9RzDUYot1R0FWgBWz3QPQtZwi5Akc20BwHUAUDEdLuQI5Wdqhx6z5ADvoxIILWIIeBEra5H/GLFpcFfq1INXNdi3bWvagUeUH4wufqZM8BvEKJcrxjgqMJoCh5U0wwnvmMtDahfIDF1XHhLboD/97hVV0MWV+qMg4/pZb0ulXXbVN1+5fsiRdfO65afOWrNVmK8+9G6BHfjsGEfBCv/MYIRige/bZZ8+BfQA6Aj5zgHICCnmuOz71aD2Hr4B85qtzgv5cdtllVWgJdXlXPDw29rfOa0FFk1Y9+mLWdwAC+xT8Yq9ivTPHVSZFl1z5EMcReo2jFXx1AcmyHbWslOAxrinoYRxO/vSnqwzu0/x86IILqmvbSj51a5vv4Ds8g+9atV1zZh1nrncpz6CB8WBMnYdnnXVWtQ4i0M4zkauscZ8x6SD1CfjZU12f/K07vlcjQj9rlnkVQyfss9fIMXfphyFAKiZ4X4s47VKvCdyoD+DqWcBa8nYOFZ5modeSz/6EcgtlGTTRrgAdwM6+SX9QALo/6nnAM/Y0LOfQbagQHlHQSA4T2qdtk5LSdz7wjrGiHLyDDjPlqzSU3+6jKAKOOeaYytuKvcw5XTKnLNMVogZfxnWf70PPrJcdAPr4IzRgg/F5OI0aBoA+Da7OaJ3DIiwbGDd8hAiuz+LQVaAtq2E2SsUYNoQNrNXGsgL+EEgQXhA+SBIn6OB7kx/Zk7YkcX1i5aLwAV8RqBCCtExoFdQChbCmBUOFAgIf7/A+z3B5RHDCeu71RPRPgZG+69Zu8iYtJAJ2BDkVA4w79arQUAD1qrQIoOK8KAEzgkEtTr4DXQh2tItFuUkIi0oQkiwxpiafqnO1jzNRCx99BkDw0Xqm2zQ89eopytDXeFWWoLoNoKsQUUhXSKUN+ikI1wqtpYo6jUGHNgRn5yuw+uRrr02vu+mmbRbX+mXL0mUrV6bNO+9cPYtKgghyBbfRsgjPBCgCKd7RlRhw9IY3vKGK3WU+oBSBLq+pgl4tzc43kxXyzGu2vJse+mifNgEN7DXM
```
Comment (nfrasser) [inline docs/examples/3dflex-custom-latent-trajectory.ipynb:268]: Looks like there was some error when this notebook ran prevents the interactive widget from showing up in the [preview](https://cryosparc-tools-git-fork-plethorachutney-cu-42f17c-structurabio.vercel.app/examples/3dflex-custom-latent-trajectory.html):

```
Unhandled message type set_device_pixel_ratio. {'device_pixel_ratio': 1.2000000476837158, 'type': 'set_device_pixel_ratio'}
```

Not sure what to do about this. Are we ok with pushing this with an omitted trajectory plot?
image

**Comment by PlethoraChutney** [comment]
Sorry about that Nick, and sorry about a million commits here! In all of these cases, both the browser and VS Code were successfully showing the plot, and the  tag looked like it was still there, but not being rendered. I opted here to just save the image and include it as an attachment instead.

**Comment by nfrasser** [comment]
No worries Rich, thanks for working out the kinks!

**Comment by nfrasser** [comment]
> Not sure if the fixed version of the dataset saving is still the right way to go though, was this written before `save_external_result()`?

Looks ok to me! Both `save_external_result()` and the `save_output()` strategy are valid. This strategy is nice because you can use `alloc_output()` to get a fresh empty dataset, which you can't do as easily with `save_external_result()`

**Pull request merged**